src/oncall/api/v0/preview.py

```diff
@@ -29,8 +29,19 @@ def on_get(req, resp, schedule_id):
     start__lt = req.get_param('start__lt', required=True)
     end__ge = req.get_param('end__ge', required=True)
     team__eq = req.get_param('team__eq', required=True)
+    roster_id = schedule['roster_id']
 
-    cursor.execute('CREATE TEMPORARY TABLE IF NOT EXISTS `temp_event` AS (SELECT * FROM `event` WHERE `start` < %s AND `end` > %s)', (start__lt, end__ge))
+    # create a temporary table with the events that include members of the team's roster
+    query = '''
+            CREATE TEMPORARY TABLE IF NOT EXISTS `temp_event` AS
+            (SELECT `event`.*
```

Comment:
Can we list all of the columns we need instead of using `*`? That'd make the usage a lot more clear.